src/main/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingResourceImpl.java

```diff
@@ -600,8 +600,11 @@ public AdjudicationSummary getAdjudicationSummary(final Long bookingId, final Lo
     }
 
     @Override
-    public Page<VisitWithVisitors<VisitDetails>> getBookingVisitsWithVisitor(final Long bookingId, final LocalDate fromDate, final LocalDate toDate, final String visitType, Pageable pageable) {
-        return bookingService.getBookingVisitsWithVisitor(bookingId, fromDate, toDate, visitType, pageable);
+    public Page<VisitWithVisitors<VisitDetails>> getBookingVisitsWithVisitor(final Long bookingId, final LocalDate fromDate, final LocalDate toDate, final String visitType, final Integer page, final Integer size) {
+        final var pageIndex = page != null ? page : 0;
```

Comment:
Not sure this is needed as annotations will set the defaults

Reply:
will write a test for that and will remove if not needed, thanks

Reply:
turns out the defaults are for the benefit of swagger ui so null checks are needed, hope that's okay :)